src/data/finsupp.lean

```diff
@@ -2,25 +2,42 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl
+-/
+
+import data.finset data.set.finite algebra.big_operators algebra.module
+
+/-!
 
-Type of functions with finite support.
+# Type of functions with finite support.
```

Comment:
```suggestion
# Type of functions with finite support
```
I don't think we're perfectly consistent about this, but it's a header, this should be preferred.